rust/src/index/vector/flat.rs

```diff
@@ -88,10 +86,8 @@ pub async fn flat_search(
 
             // TODO: use heap
             let indices = sort_to_indices(&scores, None, Some(query.k))?;
-            let batch_with_score = batch.try_with_column(
-                ArrowField::new(SCORE_COLUMN, DataType::Float32, false),
-                scores,
-            )?;
+            let batch_with_score = batch
+                .try_with_column(ArrowField::new(SCORE_COL, DataType::Float32, false), scores)?;
```

Comment:
what should the score column's field id be? how would we enforce it?

Reply:
This is still not clear to me why needs to reinforce column id here. 

Not clear to me what bug it triggers, as all python tests / rust tests passed.  Do you remember which PR # is the fix? i can look into that as well.